src/mobile-ui-react/ResizablePanel.tsx

```diff
@@ -503,3 +554,56 @@ class TouchDragHandle extends React.PureComponent<TouchDragHandleProps, TouchDra
   //   this.props.onClick?.();
   // }
 }
+
+// @todo Use the functional version of TouchDragHandle below after sufficient testing.
+// // 2024-06-24: Converted to a function due to compile errors that I couldn't understand after
```

Comment:
Remove mention of compile error since that is now resolved

Reply:
Fixed.